atlp-rwanda/commanders-rn-medica
# Feature: appointment voice call

#### What does this PR do?

This PR includes designs of voice call screens for the doctor appointment screens

#### Description of Task to be completed?

- Design appointment summary page
- Design call screen
- Design call ended screen 

#### How should this be manually tested?

Navigate to the home page (after login). Click on appointment tab. On upcoming tab, click on the phone icon on any doctor card. My appointment screen should display and by clicking on Start you are navigated to the call screen with controls, wait 5 seconds for the state to change and click on end call button. You are navigated to the session ended page.

#### Any background context you want to provide?

Clicking on the phone icon works only on upcoming tab, on other tabs the user will be redirected to other pages than starting a call.

#### What are the relevant pivotal tracker/Trello stories?

N/A

#### Screenshots (if appropriate)

<img width="230" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/c55470d3-b306-472d-b2ef-29a5b8733731" />
<img width="230" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/205ed047-9df8-4d3a-aaec-559ee2dc04cf" />
<img width="230" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/ee4d2a50-b5de-4afd-aba7-9e66f27d4d2a" />

#### Questions:

N/A
